plugins/packages/polygon/main.go

```diff
@@ -186,7 +294,82 @@ func Migrate(db *sql.DB, chainid uint64) error {
 		if _, err := db.Exec("UPDATE bor.migrations SET version = 1;"); err != nil {
 			return err
 		}
+		if _, err := db.Exec(`CREATE INDEX bor.logsTxHash ON bor_logs(transactionHash)`); err != nil {
+			log.Error("bor_receiptBlock CREATE INDEX error", "err", err.Error())
+		}
+		if _, err := db.Exec(`CREATE INDEX bor.logsBkHash ON bor_logs(blockHash)`); err != nil {
+			log.Error("bor_receiptBlock CREATE INDEX error", "err", err.Error())
+		}
+	}
+	if schemaVersion < 2 {
+		rows, _ := db.QueryContext(context.Background(), "SELECT parentHash, uncleHash, root, txRoot, receiptRoot, bloom, difficulty, number, gasLimit, gasUsed, `time`, extra, mixDigest, nonce, baseFee from blocks.blocks where coinbase = X'00';")
+		defer rows.Close()
+
+		for rows.Next() {
+			var bloomBytes, parentHash, uncleHash, root, txRoot, receiptRoot, extra, mixDigest, baseFee []byte
+			var number, gasLimit, gasUsed, time, difficulty uint64
+			var nonce int64
+			err := rows.Scan(&parentHash, &uncleHash, &root, &txRoot, &receiptRoot, &bloomBytes, &difficulty, &number, &gasLimit, &gasUsed, &time, &extra, &mixDigest, &nonce, &baseFee)
+			if err != nil {log.Info("sacn error", "err", err.Error())}
+			
+			logsBloom, _ := decompress(bloomBytes)
+			if err != nil {
+				log.Info("Error decompressing data", "err", err.Error())
+			}
+			
+			var lb [256]byte
+			copy(lb[:], logsBloom)
+			var bn [8]byte
+			binary.BigEndian.PutUint64(bn[:], uint64(nonce))
+			dif := new(big.Int).SetUint64(difficulty)
+			num := new(big.Int).SetUint64(number)
+			hdr := &evm.Header{
+				ParentHash: bytesToHash(parentHash),
+				UncleHash: bytesToHash(uncleHash),
+				Root: bytesToHash(root),
+				TxHash: bytesToHash(txRoot),
+				ReceiptHash: bytesToHash(receiptRoot),
+				Bloom: lb,
+				Difficulty: dif,
+				Number: num,
+				GasLimit: gasLimit,
+				GasUsed: gasUsed,
+				Time: time,
+				Extra: extra,
+				MixDigest: bytesToHash(mixDigest),
+				Nonce: bn,
+			}
+			if len(baseFee) > 0 {
+				hdr.BaseFee = new(big.Int).SetBytes(baseFee)
+			}
+			var miner common.Address
+			if len(hdr.Extra) == 0 {
+				miner = common.HexToAddress("0x0000000000000000000000000000000000000000")
```

Comment:
I would just set this to `common.Address{}`, which is equivalent, but without having to parse a hex string to get it.